pkg/kubectl/bind-apiservice/plugin/servicebindings_test.go

```diff
@@ -0,0 +1,756 @@
+/*
+Copyright 2022 The Kube Bind Authors.
```

Comment:
```suggestion
Copyright 2023 The Kube Bind Authors.
```

Reply:
Corrected this in 926bb5993775516f6dbfe28787228a9b11d95f92.